apps/storybook/package.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "storybook",
```

Comment:
as per our naming scheme it should be "@umami/storybook"
or "@umami/storybook-web" because we might want to have it also for other apps too

Reply:
I think one storybook is enough, we can compose components from other libraries